packages/backend-modules/republik-crowdfundings/migrations/sqls/20230710172000-package-option-fixed-price-up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE "public"."packageOptions"
+  ADD COLUMN "payMoreSuggestion" boolean
```

Comment:
Don't these need a default value of `FALSE`, otherwise GraphQL will complain about returning a `null` value for non-optional fields?

Reply:
Also: do we need to populate these fields for existing packageOptions (we might not if there's a default value)? Shouldn't this be part of the migration?